### DISCUSSION TITLE
wrapping JSF tab elements with HTML nav element.

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#2302] [PrimeFaces] [answered]
I would like to mark the tab elements of a tabView as region navigation (nav). If I surround the tab elements with nav, PrimeFaces ignores it and if I surround the entire tabView with nav, the content also belongs to the navigation region.

Do you have any idea how I can solve this?

**Comment by melloware** [comment]
are you trying to mark the tab headers or the contents of the tab as a `region`?

**Reply by hgfrey** [reply]
Yes.

The markup should look like this:

![image](https://github.com/user-attachments/assets/64a0c0ec-9ce8-4484-b868-065efb2f5a88)

**Reply by melloware** [reply]
@hgfrey i am not sure where you are getting `navigation` and `region` from that is not proper ARIA for Tabview. See: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/

**Reply by hgfrey** [reply]
Our head of accessibility says: 
You are basically right, since not every tab view is used as a navigation element.
But in our case, we use it to centrally control the form content. In the example there are index tabs.

**Reply by melloware** [reply, marked as answer]
i think the core TabView component would have to be changed to render the HTML you want but to me that feels wrong as ARIA says a TabView is a TabView and we are following the ARIA spec.
